common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,63 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 getVector3(const double x, double y, double z)
```

Comment:
I feel it's better to use `create` or `make`.

I originally used `create` functions in `tier4_autoware_utils`, but considering it's a simple process, is `make` better?
https://ell.stackexchange.com/questions/76985/create-vs-generate-vs-make

But since consistency is also important, so how about using `create` for now and renaming it in the refactoring phase?

```suggestion
inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
```